README.md

```diff
@@ -321,6 +321,13 @@ Run the command `heroku pg:backups:download -o {app-name}.dump -a {app-name}` wh
 
 Then to import the data into your new AWS RDS database run `pg_restore -v -h {rds-endpoint} -U {username} -d quodl {app-name}.dump` and enter the instance master user's password when prompted. The RDS endpoint can be found by selecting your instance on the RDS dashboard and scrolling down to the `Connect` box (as shown in the section above).
 
+Your data should now have been migrated to RDS, the only thing left to do is switch the databases over. If you're using the `DATABASE_URL` config variable on Heroku, you need to detach your current database before you can change it. Unfortunately, detaching the database destroys it and everything in it. To avoid this, you can first by first running the command `heroku addons:attach {database-name} --as backup_db -a {app-name}`. To find your database name, run `heroku addons` on your command line; The database name is in brackets after `heroku-postgresql`. (Here, it's `postgresql-metric-95269`.)
```

Comment:
@Danwhy that's a _lot_ of text for a single line ... see: https://github.com/dwyl/style-guide/issues/9